apps/data-types.md

```diff
@@ -84,6 +84,22 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | pageTitle | string | The page title of this category. |
 | description | string | The description of the category. |
 
+## contactInformation
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| name | string | The name that appears on the contact information page, e.g. Contact Us. |
+| title | string | The name of the contact information page, that appears on the browser tab. |
+| navigationCaption | string | The name of the contact information page that appears in the navigation bar. |
+| shortDescription | string | Additional short information that can be given to e.g. better explain what's on the contact information page. |
+| description | string | Additional information that can be added to the contact information page, e.g. tax identification number or bank account. |
+| company | string | The name of the shop. |
+| contactPerson | string | The contact person for the shop, usually the shop owner. |
+| contactPersonJobTitle | string | The job title of the contact person. |
+| address | string | The postal address of the shop. |
+| phone | string | The phone number of the shop. |
+| email | string | The email address of the shop. |
+
 ## contentPageSummary
```

Comment:
it is an old issue, but i just found it: *contentPageSummary* should contain the attributes:
- contactInformation
- privacyPolicy
- ...
which are of type *contentPageSummaryItem* which just have the attribute *name*